index.html

```diff
@@ -3,10 +3,144 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <link rel="stylesheet" href="/css/Nabvar.css">
+  <link rel="stylesheet" href="/css/burgerMenuWindows.css">
+  <link rel="stylesheet" href="/css/dinamicSection.css">
+  <title>CapstoneFernando</title>
 </head>
 <body>
-  
+  <div class="upVar">
+    <ul class="Uplist">
+      <a href=""><img src="/images/facebook.png" class="logFaceAndTwit" srcset=""></a>
+      <a href=""><img src="/images/twitter.png" class="logFaceAndTwit" srcset=""></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </ul>
+  </div>
+  <header class="header">
+    <div>
+      <nav class="mobile-navbar">
+        <button><img src="./images/mobile_menu 1.png" alt="burgerMenu" onclick ="opneNave();" class="image-burger-menu"></button>
+      </nav>
+    </div>
+    
+    <div class="container-logo">
+      <a href="index.html"><img src="./images/LogoUrban.png" alt= My Logo class="logo"></a>
+    </div>
+      <nav class="navbar">
```

Comment:
- [ ] When I tested your mobile menu and scrolled through your About page, I noticed that a few images are overlapping with your menu. To enhance the professional appearance of your project, it would be awesome if you could address this issue and make sure the images and menu are displayed neatly without any overlaps.

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/3dd57beb-f4a7-4c3f-b581-761b41e607d0)